feat(changelog): object-storage-added-uncomplete-multipart-uploads-are-n 2024-09-16 (#3701)
scaleway/docs-content: changelog/september2024/2024-09-16-object-storage-added-uncomplete-multipart-uploads-are-n.mdx

---

> title: Uncomplete multipart uploads are now visible in Console!

Reviewer: ```suggestion
title: Incomplete multipart uploads are now visible in the Scaleway console!
```

status: added
author:
    fullname: 'Join the #object-storage channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2024-09-16
category: storage
product: object-storage
---

> It is now possible to list and abort uncomplete multipart uploads from the Console. Please follow this how-to guide to navigate to your uncomplete multipart uploads: https://www.scaleway.com/en/docs/storage/object/how-to/abort-incomplete-mpu
> 

Reviewer: ```suggestion
It is now possible to list and abort incomplete multipart uploads from the Scaleway console. Follow the [dedicated documentation](/storage/object/how-to/abort-incomplete-mpu) to navigate to your incomplete multipart uploads.
```
